tests/package.json

```
@@ -0,0 +1,14 @@
+{
+    "scripts": {
+        "test:e2e": "cucumber-js --require cucumber.conf.js --require e2e/stepDefinitions/**/*.js --format @cucumber/pretty-formatter"
```

you can use shorthand options if the command gets too long
```suggestion
        "test:e2e": "cucumber-js -r cucumber.conf.js -r e2e/stepDefinitions/**/*.js -f @cucumber/pretty-formatter"
```